posts/2023/mfdataset.ipynb

```diff
@@ -0,0 +1,5675 @@
+{
```

Comment:
<p>Above it mentions a chunk size of 16 lat and 32 lon, but the code here is chunking in time only?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>fixed now. thanks!</p>